core/domain/auth_services_test.py

```diff
@@ -81,6 +81,16 @@ def test_get_all_profiles_for_parent_user_id_returns_all_profiles(self):
             [auth_models.UserAuthDetailsModel.get(self.profile_user_1_id),
              auth_models.UserAuthDetailsModel.get(self.profile_user_2_id)])
 
+    def test_establish_auth_session(self):
+        request = webapp2.Request.blank('/')
+        response = webapp2.Response('/')
+        self.assertIsNone(auth_services.establish_auth_session(
```

Comment:
I don't really know what this is testing... If you actually want to establish that an auth session has indeed been created, shouldn't you be checking that e.g. the auth cookies are set?

Reply:
I'm actually still working on auth_services_test.py. Should I unchange those lines for this PR for now?